### PR TITLE
Use make tu lunch the upgrade process

### PR DESCRIPTION
**Pull request by sbrunner**
Also remove deprecated documentation

Related to #1488

### CODE REVIEW

File: doc/integrator/upgrade_application.rst [hunk 1]
```diff
@@ -28,6 +28,29 @@ Where ``<makefile>`` is your user make file (``<user>.mk``).
 
 Easy upgrading an application
 -----------------------------
+
+In the ``setup.py`` be sure not to specify a ``c2cgeoportal`` version,
+because it will prevent the installation of the new ``c2cgeoportal`` egg.
+
+It consider that your makefile is named ``<instanceid>.mk``, if it's not the case
```
Comment (elemoine) [inline doc/integrator/upgrade_application.rst]: It = c2ctool? I think you can repeat c2ctool here to avoid confusion.

Comment (elemoine) [inline doc/integrator/upgrade_application.rst]: consider -> considers

Comment (elemoine) [inline doc/integrator/upgrade_application.rst]: makefile -> Makefile

File: doc/integrator/upgrade_application.rst [hunk 2]
```diff
@@ -28,6 +28,29 @@ Where ``<makefile>`` is your user make file (``<user>.mk``).
...
+
+   You should add ``UPGRADE_ARGS ?= --windows`` in your ``<package>.mk`` file.
+
+Than just run:
+
+.. prompt:: bash
+
+   VERSION=<target_version> make -f <makefile> upgrade
+
+Where ``<makefile>`` is your user make file (``<user>.mk``),
```
Comment (elemoine) [inline doc/integrator/upgrade_application.rst]: make file -> Makefile

**Comment by elemoine** [comment]
I just added a few minor comments. Please merge when addressed.

File: doc/integrator/upgrade_application.rst (hunk 1 above)
Comment (sbrunner) [inline doc/integrator/upgrade_application.rst]: It's not c2ctool, but the makefile, then it replace it by We :-)

**Comment by sbrunner** [comment]
Thanks @elemoine  :-)

**Pull request merged**